scripts/globals/caskets.lua

```diff
@@ -219,6 +219,4260 @@ local casketItems =
+    {
+        regionalItems = {15927}, -- Pinwheel Belt
+        temps =
+        {
+        ----------------------------------
+        --| Weight | Item ID |   Name  |--
+        ----------------------------------
+            {1200, 4114}, -- Potion +2
+            { 950, 4115}, -- Potion +3
+            {1000, 4116}, -- Hi-Potion
+            {1100, 4129}, -- Ether +1
+            { 960, 4130}, -- Ether +2
+            { 450, 4131}, -- Ether +3
+            { 250, 4136}, -- Super Ether
+            { 750, 4145}, -- Elixer
+            { 820, 4148}, -- Antidote
+            { 730, 4150}, -- Eye Drops
+            { 850, 4151}, -- Echo Drops
+            { 650, 4155}, -- Remedy
+            {1020, 4164}, -- Prism Powder
+            { 940, 4165}, -- Silent Oil
+            { 450, 4202}, -- Daedalus Wing
+            { 310, 4206}, -- Catholicon
+            { 240, 4302}, -- Pamama Au Lait
+            { 660, 4424}, -- Melon Juice
+        },
+        items =
+        {
+        ----------------------------------
+        --| Weight | Item ID |   Name  |--
+        ----------------------------------
+            {4600,   642}, -- Zinc Ore
+            {1250,   689}, -- Lauan Log
+            { 600,   693}, -- Walnut Log
+            {3250,   736}, -- Silver Ore
+            {2900,   835}, -- Flax Flower
+            {4800,   847}, -- Bird Feather
+            {1600,   898}, -- Chicken Bone
+            {2020,   914}, -- Mercury
+            {5000,   936}, -- Rock Salt
+            { 540,   947}, -- Firesand
+            {3540,  1108}, -- Sulfur
+            {4750,  1211}, -- Handful of Bronze Bolt Heads x33
+            {2900,  4113}, -- Potion +1
+            {2600,  4128}, -- Ether
+            { 575,  4148}, -- Antidote
+            {3150,  4151}, -- Echo Drops
+            {1010,  4373}, -- Woozyshroom
+            { 200, 16643}, -- Battleaxe
+            { 520, 12455}, -- Beetle Mask
+            { 240, 12583}, -- Beetle Harness
+            { 680, 12711}, -- Beetle Mittens
+            {1200, 12835}, -- Beetle Subligar
+            { 990, 12967}, -- Beetle Leggings
+            { 880, 12865}, -- Black Slacks
+            {1530, 12966}, -- Bone Leggings
+            {1760, 12454}, -- Bone Mask
+            {1900, 12710}, -- Bone Mittens
+            {1870, 12834}, -- Bone Subligar
+            { 310, 13584}, -- Cotton Cape
+            { 595, 13588}, -- Dhalmel Mantle
+            {1580, 12985}, -- Holly Clogs
+            { 275, 16473}, -- Kukri
+            {1680, 12729}, -- Linen Cuffs
+            {1460, 12857}, -- Linen Slops
+            {2620, 12697}, -- Lizard Gloves
+            { 645, 12441}, -- Lizard Helm
+            { 835, 12953}, -- Lizard Ledelsens
+            { 610, 12825}, -- Lizard Trousers
+            { 455, 12473}, -- Poet's Circlet
+            { 745, 12993}, -- Sandals
+            {2165, 12737}, -- White Mitts
+        }
+    },
+    [dsp.zone.BATALLIA_DOWNS] =
+    {
+        regionalItems = {18451}, -- Mokusa
```

Comment:
I'm just curious why regionalItems exists rather than just putting the rare ex item in with the other non temps. I don't think retail makes a distinction its just weighted to be low rate. not a complaint/thing to fix atm tho.

Reply:
I noticed that i was getting too many instances of a whole chest with nothing but regional items, this was my attempt to reduce this from happening.

Reply:
k we'll figure that out later down the road. It may be better to have them like this anyway just because this way they are easy to spot and if someone wanted to globally adjust just those, they can hit them all at once instead of in each zone table. So from a not giving any cares what retail does standpoint, its handy.